ansible/vars/topo_t0-u254d2.yml

```diff
@@ -0,0 +1,5370 @@
+    ARISTA226T1:
+      vlans:
+        - 227
+      vm_offset: 225
+    ARISTA227T1:
+      vlans:
+        - 228
+      vm_offset: 226
+    ARISTA228T1:
+      vlans:
+        - 229
+      vm_offset: 227
+    ARISTA229T1:
+      vlans:
+        - 230
+      vm_offset: 228
+    ARISTA230T1:
+      vlans:
+        - 231
+      vm_offset: 229
+    ARISTA231T1:
+      vlans:
+        - 232
+      vm_offset: 230
+    ARISTA232T1:
+      vlans:
+        - 233
+      vm_offset: 231
+    ARISTA233T1:
+      vlans:
+        - 234
+      vm_offset: 232
+    ARISTA234T1:
+      vlans:
+        - 235
+      vm_offset: 233
+    ARISTA235T1:
+      vlans:
+        - 236
+      vm_offset: 234
+    ARISTA236T1:
+      vlans:
+        - 237
+      vm_offset: 235
+    ARISTA237T1:
+      vlans:
+        - 238
+      vm_offset: 236
+    ARISTA238T1:
+      vlans:
+        - 239
+      vm_offset: 237
+    ARISTA239T1:
+      vlans:
+        - 240
+      vm_offset: 238
+    ARISTA240T1:
+      vlans:
+        - 241
+      vm_offset: 239
+    ARISTA241T1:
+      vlans:
+        - 242
+      vm_offset: 240
+    ARISTA242T1:
+      vlans:
+        - 243
+      vm_offset: 241
+    ARISTA243T1:
+      vlans:
+        - 244
+      vm_offset: 242
+    ARISTA244T1:
+      vlans:
+        - 245
+      vm_offset: 243
+    ARISTA245T1:
+      vlans:
+        - 246
+      vm_offset: 244
+    ARISTA246T1:
+      vlans:
+        - 247
+      vm_offset: 245
+    ARISTA247T1:
+      vlans:
+        - 248
+      vm_offset: 246
+    ARISTA248T1:
+      vlans:
+        - 249
+      vm_offset: 247
+    ARISTA249T1:
+      vlans:
+        - 250
+      vm_offset: 248
+    ARISTA250T1:
+      vlans:
+        - 251
+      vm_offset: 249
+    ARISTA251T1:
+      vlans:
+        - 252
+      vm_offset: 250
+    ARISTA252T1:
+      vlans:
+        - 253
+      vm_offset: 251
+    ARISTA253T1:
+      vlans:
+        - 254
+      vm_offset: 252
+    ARISTA254T1:
+      vlans:
+        - 255
+      vm_offset: 253
+  DUT:
+    vlan_configs:
+        default_vlan_config: one_vlan_per_intf
+        one_vlan_per_intf:
+        Vlan1000:
+            id: 1000
+            intfs: [0]
+            prefix_v6:  fc00:c:c:0001::/64
+            tag: 1000
+        Vlan1001:
+            id: 1001
+            intfs: [1]
+            prefix_v6:  fc00:c:c:0002::/64
+            tag: 1001
+
+configuration_properties:
+  common:
+    dut_asn: 4200000000
+    dut_type: ToRRouter
+    swrole: leaf
+    podset_number: 200
+    tor_number: 16
+    tor_subnet_number: 2
+    max_tor_subnet_number: 16
+    tor_subnet_size: 128
+    spine_asn: 4200200000
+    leaf_asn_start: 4200100000
+    tor_asn_start: 4200000000
+    failure_rate: 0
+    nhipv6: FC0A::FF
+
+configuration:
+  ARISTA01T1:
+    properties:
+    - common
+    bgp:
+      router-id: 0.12.0.3
+      asn: 4200100000
+      peers:
+        4200000000:
+          - fc00:a::5
+    interfaces:
+      Loopback0:
+        ipv6: fc00:c:c::3/64
```

Comment:
IPv6 loopback should still be /128. 
for the p2p IPv6 address which is used for Ethernet1, it should be /64 as Jing suggested.   cc'ing @r12f in case if he has any input/comment on this one.

Reply:
discussed in the sync meeting, the step needs to be set to /64, subnet needs to be set to /128.